src/api/flaskr/service/user/common.py

```
@@ -421,10 +454,161 @@ def verify_sms_code(app: Flask, user_id, phone: str, chekcode: str) -> UserToken
+        if course_count != 1:
+            return
+
+        # Get the only course
+        course = AICourse.query.first()
+        if course:
+            # The creator of the updated course
+            course.created_user_id = user_id
+            db.session.commit()
+
+
+def init_first_course(app: Flask, user_id: str):
```

Ruff (0.8.2)

Redefinition of unused `init_first_course` from line 554

(F811)